packages/web3-eth-accounts/src/local_storage_wallet.ts

```diff
@@ -7,7 +7,7 @@ import {
 
 type BrowserError = { code: number; name: string };
 
-export class LocalStorageWallet<
+export class Wallet<
```

Comment:
Existing implementation is better matching with the name `LocalStorageWallet`. If want to keep old name for the sake of 1.x then export it in the `index.ts` instead with a deprecation warning.

Reply:
lets closely follow 1.x